packages/config/config/devices/0x0287/iblindsv3.json

```diff
@@ -0,0 +1,105 @@
+{
+	"manufacturer": "HAB Home Intelligence LLC",
+	"manufacturerId": "0x0287",
+	"label": "IBLINDSV3",
+	"description": "iblinds V3",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0071"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Close Interval",
+			"description": "Auto Calibration Tightness\n\t  Lower value = tighter\n\t  Higher value = looser\n\t  MUST BE RECALIBRATED\n\t  Range:16-32\n      Default: 22",
```

Comment:
remove unnecessary \n and whitespace

Reply:
same goes for the other descriptions

Reply:
I would use commit suggestions so we can edit from here

Reply:
Also I think this needs to use partial parameters right?